### PR TITLE
Fix/responsive main page

### PR DESCRIPTION
**Pull request by mv-v-7**
https://trello.com/c/7YKDIJ8H
https://trello.com/c/NndDkNrR

### CODE REVIEW

File: client/src/components/auth/LayoutMainPage.vue [hunk 1]
```diff
@@ -92,9 +109,11 @@ export default {
 <style>
 html {
     overflow-y: auto !important;
+    font-size: 10px;
 }
 </style>
 <style scoped>
+/*block styling*/
```
Comment (lenchvolodymyr) [inline client/src/components/auth/LayoutMainPage.vue:116]: do not leave comments, no one needs it

File: client/src/components/auth/LayoutMainPage.vue [hunk 2]
```diff
@@ -110,64 +129,89 @@ html {
...
     .first-column-in-secondary {
         display: none;
     }
     .office-worker-image {
-        margin-top: 10px;
+        margin-top: 5vmin;
         min-height: 50vmin;
     }
+    .logo-primary-div {
+        display: block !important;
```
Comment (lenchvolodymyr) [inline client/src/components/auth/LayoutMainPage.vue:165]: I'm not sure you need !important here, above you do display: none, and it works, why doesn't it here?

File: client/src/components/auth/LayoutMainPage.vue [hunk 3]
```diff
@@ -110,64 +129,89 @@ html {
...
+    }
+    .logo-secondary-div {
+        display: none;
+    }
+    .place-for-form-vcol {
+        margin-top: 9vh !important;
```
Comment (lenchvolodymyr) [inline client/src/components/auth/LayoutMainPage.vue:171]: the same here

**Comment by mv-v-7** [comment]
All done

**Pull request merged**